api/v1/tests.py

```diff
@@ -1,3 +1,68 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse_lazy
+
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from fixtureless.factory import create
+
+from recipes.models import Recipe
+
+
+class TestRecipeApi(APITestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create(username='test_user')
+        self.recipe = create(Recipe, {'title': 'testing'})
+
+    def test_recipe_list(self):
+        """
+        Verify GET returns Recipe list
+        """
+        response = self.client.get(reverse_lazy('api:v1:recipe-list'))
```

Comment:
since the url is used in every test' and to eliminate cruft from the tests themselves that make them less clear, assign the url to an instance variable in the setup, ie `self.recipe_list_url = reverse_lazy('api:v1:recipe-list')`